src/api/Feed.js

```diff
@@ -1344,6 +1352,74 @@ class Feed extends Base {
         }
     }
 
+    /**
+     * Fetches app activities for a file
+     * @param {BoxItemPermission} permissions - Permissions to attach to the app activity items
+     *
+     * @return {Promise} - the feed items
+     */
+    fetchAppActivity(permissions: BoxItemPermission): Promise<?AppActivityItems> {
+        this.appActivityAPI = new AppActivityAPI(this.options);
+
+        return new Promise(resolve => {
+            this.appActivityAPI.getAppActivity(
+                this.id,
+                permissions,
+                resolve,
+                this.fetchFeedItemErrorCallback.bind(this, resolve),
+            );
+        });
+    }
+
+    /**
+     * Deletes an app activity item.
+     *
+     * @param {BoxItem} file - The file to which the app activity belongs to
+     * @param {string} appActivityId - The app activity item id to delete
+     * @param {Function} successCallback - the function which will be called on success
+     * @param {Function} errorCallback - the function which will be called on error
+     * @return {void}
+     */
+    deleteAppActivity = (
+        file: BoxItem,
+        appActivityId: string,
+        successCallback: Function,
+        errorCallback: ErrorCallback,
+    ): void => {
+        this.appActivityAPI = new AppActivityAPI(this.options);
+        if (!file.id) {
+            throw getBadItemError();
+        }
+
+        const { id } = file;
```

Comment:
Should this be moved to the first line so it can be used in the `!file.id` check?